### ISSUE TITLE
Can we rename `--vue-indent-script-and-style` to `--indent-script-and-style` for all SFC files like `svelte`?

### ISSUE THREAD

**Issue by JounQin** [prettier/prettier#13259]
Can we rename `--vue-indent-script-and-style` to `--indent-script-and-style` for all SFC files like `svelte`?

[prettier-plugin-svelte](https://github.com/sveltejs/prettier-plugin-svelte#svelte-indent-script-and-style) has to have a new `--svelte-indent-script-and-style` option, but they could be unified, actually.

_Originally posted by @JounQin in https://github.com/prettier/prettier/issues/13142#issuecomment-1208280895_

**Comment by fisker** [comment]
I remember there are also people looking for options for indentation of head  and body in html, maybe we can also support in one flag?

**Comment by JounQin** [comment]
> Didn't know this was discussed [#3888 (comment)](https://github.com/prettier/prettier/issues/3888#issuecomment-473973916), and we don't support other SFC files in core. So I guess we should close this.

I don't think so, it's major version, although we're not SFC except officially, but it still can be standard.
